fixtures/1.json

```
@@ -0,0 +1,5 @@
+{
+	"identifiers": "['1', '3']",
```

Hmm, this doesn't look right to me. I would think that these would be passed as arrays, rather than string representations of arrays:
```suggestion
	"identifiers": ["1", "3"],
```
It might also be worth figuring out whether those IDs will be strings or numbers.

---

The result of
```for obj in Bag.objects.all():
            print('"identifiers": {}, "start_date": {}, "end_date": {}'.format(obj.rights_id, obj.start_date, obj.end_date))
```
in the Zorya tests is:
```"identifiers": ['9', '16'], "start_date": 1911-06-12, "end_date": 1966-07-21
"identifiers": ['9', '13'], "start_date": 1949-04-10, "end_date": 1980-06-09
```
I'm not sure of the best way of figuring out what the `post` function from the `requests` module is creating?

---

OK, it looks to me like:
- The rights_id field in Zorya is a JSONField, which means that the data that gets passed in will be stored as JSON.
- What's coming back from the Zorya tests would likely be serialized in JSON as what I described above. 
- If you want to see what Zorya is sending I'd print out `resp.data` or maybe `resp.body` in the `retrieve_rights()` method.